src/main/java/com/haxwell/apps/quizki/controllers/VerifyCredentialsController.java

```diff
@@ -50,7 +55,8 @@
 	 * @return
 	 * @throws Exception
 	 */
-	@GetMapping
+	//@GetMapping
```

Comment:
You can remove the commented out @GetMapping .. Its the same as the expanded version you added with RequestMapping....